src/google/cloud/ndb/model.py

```diff
@@ -4082,33 +4082,95 @@ def non_transactional(*args, **kwargs):
     raise NotImplementedError
 
 
-def get_multi_async(*args, **kwargs):
-    raise NotImplementedError
+def get_multi_async(keys, **options):
+    """Fetches a sequence of keys.
 
+    Args:
+        keys (Sequence[:class:`~google.cloud.ndb.key.Key`]): A sequence of keys.
+        **options (Dict[str, Any]): The options for the request. For example,
+            ``{"read_consistency": EVENTUAL}``.
+    Returns:
+        List[:class:`~google.cloud.ndb.tasklets.Future`]: List of futures.
+    """
+    return [key.get_async(**options) for key in keys]
 
-def get_multi(*args, **kwargs):
-    raise NotImplementedError
 
+def get_multi(keys, **options):
+    """Fetches a sequence of keys.
 
-def put_multi_async(*args, **kwargs):
-    raise NotImplementedError
+    Args:
+        keys (Sequence[:class:`~google.cloud.ndb.key.Key`]): A sequence of keys.
+        **options (Dict[str, Any]): The options for the request. For example,
+            ``{"read_consistency": EVENTUAL}``.
+    Returns:
+        List[Union[:class:`~google.cloud.ndb.model.Model`, :data:`None`]]: List
+            containing the retrieved models or None where a key was not found.
+    """
+    return [key.get(**options) for key in keys]
```

Comment:
I think this will just get each entity in series, which doesn't take any advantage of batching or parallelism. Generally, if there's an X and an X_async, X calls X_async and then waits for the result.

Reply:
Should still fire all the gets asynchronously, then wait for the result:

futures = [key.get_async(**options) ... yada yada]
return [future.result() for future in futures]